core/chain/security/blockaid/scanTransaction.ts

```
@@ -0,0 +1,58 @@
+import { getBlockaidTxScanUrl } from './txScanUrl'
+import { BlockaidError, BlockaidScanResult } from './types'
+
+export async function validateTransaction({
+  chain,
+  accountAddress,
+  rawTx,
+}: {
+  chain: BlockaidSupportedChains
+  accountAddress?: string
+  rawTx?: string
+}): Promise<{ error: BlockaidError } | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)